tests/pytests/unit/modules/test_gpg.py

```diff
@@ -1039,3 +1044,76 @@ def test_gpg_decrypt_message_with_gpg_passphrase_in_pillar(gpghome):
                     gnupghome=str(gpghome.path),
                 )
                 assert ret["res"] is True
+
+
+@pytest.fixture(params={})
+def import_result_mock(request):
```

Comment:
```suggestion
def _import_result_mock(request):
```
Sorry, I should have included this change in the previous review comment.
Same reason as before.